pom.xml

```diff
@@ -201,13 +212,13 @@
                     </execution>
 
                     <execution>
-                        <id>grunt build</id>
+                        <id>grunt build/serve</id>
```

Comment:
Das Argument "serve" ist bei uns dynamisch. Wir haben ja zwei Maven Profile: "prod" und "dev". Nur in dev soll der Grunt Task "serve" sein, ansonsten einfach nur "build".
Von daher bitte das serve wieder streichen, wir haben das in den entsprechenden Profilen schon...